cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/AppLifecycleHttpHandlerTest.java

```diff
@@ -328,35 +338,39 @@ public void testOwnerInHeaders() throws Exception {
     deleteNamespace(NamespaceId.DEFAULT.getNamespace());
   }
 
+  /*
+  * Inroduced in LCM: Both the versioned and non-versioned apis deploy a versioned application.
+  * */
   @Test
+  @Ignore
```

Comment:
why ignore?

Reply:
Because this test isn't really needed anymore since we don't have a "non versioned app"